packages/salesforcedx-vscode-core/src/commands/forceConfigSet.ts

```diff
@@ -4,33 +4,72 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+import { ConfigUtil, LibraryCommandletExecutor } from '@salesforce/salesforcedx-utils-vscode/out/src';
+import { Row, Table } from '@salesforce/salesforcedx-utils-vscode/out/src/output';
+import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/src/types';
```

Comment:
The three above imports should be updated to just import from `@salesforce/salesforcedx-utils-vscode`